### PR TITLE
Basic windows support

### PR DESCRIPTION
**Pull request by ErikBjare**

### CODE REVIEW

File: aw_watcher_window/main.py [hunk 1]
```diff
@@ -54,6 +54,7 @@ def main():
     while True:
         try:
             current_window = get_current_window()
+            logger.debug(current_window)
```
Comment (johan-bjareholt) [inline aw_watcher_window/main.py]: Remove?

File: aw_watcher_window/windows.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+def get_app_path(hwnd) -> Optional[str]:
+    """Get application path given hwnd."""
+    path = None
+    _, pid = win32process.GetWindowThreadProcessId(hwnd)
+    for p in c.query('SELECT ExecutablePath FROM Win32_Process WHERE ProcessId = %s' % str(pid)):
+        path = p.ExecutablePath
+        break
+    return path
+
+def get_app_name(hwnd) -> Optional[str]:
```
Comment (johan-bjareholt) [inline aw_watcher_window/windows.py]: Unused?

File: aw_watcher_window/main.py (hunk 1 above)
Comment (ErikBjare) [inline aw_watcher_window/main.py]: I might leave it there, if people run it with `--verbose` that's the kind of output they'd expect.

File: aw_watcher_window/windows.py (hunk 2 above)
Comment (ErikBjare) [inline aw_watcher_window/windows.py]: Not `get_app_name`, but yeah, `get_app_path` is. But might be useful in the future, so keeping it there.

**Comment by ErikBjare** [comment]
Works alright, merging.

**Pull request merged**